### PR TITLE
some improvements, refactoring, add tart virtualized gtihub-runners

### PR DESCRIPTION
**Pull request by DavHau**
This adds a deploy app for the linux host as well, that does not depend on nixos-rebuild and therefore will work on macos

### CODE REVIEW

File: modules/darwinConfigurations.macos-01/arm64-setup-notes.md [hunk 1]
```diff
@@ -23,3 +23,13 @@ This will provide compatibility with x86_64-darwin
 ```command
 softwareupdate --install-rosetta --agree-to-license
 ```
+
+(for VM support, see tart.run)
```
Comment (steveej) [inline modules/darwinConfigurations.macos-01/arm64-setup-notes.md]: is tart already used somwhere?

File: modules/darwinConfigurations.macos-01/default.nix [hunk 2]
```diff
@@ -1,14 +1,14 @@
 { self, lib, inputs, config, ... }: {
   perSystem = { config, self', inputs', pkgs, ... }: {
-    apps."deploy-${(import ./attrs.nix).hostName}" = {
+    apps."deploy-macos-01" = {
```
Comment (steveej) [inline modules/darwinConfigurations.macos-01/default.nix]: can we use a variable for DRY?

**Review by steveej — COMMENTED** [review]
please update the top-level README with the updated deployment instructions :pray:

File: modules/darwinConfigurations.macos-01/arm64-setup-notes.md (hunk 1 above)
Comment (DavHau) [reply modules/darwinConfigurations.macos-01/arm64-setup-notes.md]: not yet, I just added some hints how to install it.

File: modules/darwinConfigurations.macos-01/default.nix (hunk 2 above)
Comment (DavHau) [reply modules/darwinConfigurations.macos-01/default.nix]: fixed via https://github.com/holochain/holochain-infra/pull/5/commits/484666de5207337180e9bfa70280883d4fa0e798
Those apps are now automatically generated for all hosts in the repo

**Comment by DavHau** [comment]
> please update the top-level README with the updated deployment instructions pray

done

File: modules/darwinConfigurations.macos-01/arm64-setup-notes.md (hunk 1 above)
Comment (DavHau) [reply modules/darwinConfigurations.macos-01/arm64-setup-notes.md]: All runners are now running via tart

**Review by steveej — APPROVED** [review]

**Pull request merged**